AchievementLocations_data.lua

```diff
@@ -271,6 +287,13 @@ A{"BurningSteppes", 4901, criterion=13709} -- Burning Steppes
 -- Exploration: Field Photographer
 A{"BurningSteppes", 9924, 0.200, 0.330, criterion=27968} -- Blackrock Mountain
 
+-- Exploration: Pest Control
+A{"BurningSteppes", 2556, 0.258, 0.250, criterion="Fire Beetle"}
+A{"BurningSteppes", 2556, 0.168, 0.218, criterion="Fire Beetle"}
+A{"Dalaran", 2556, 0.644, 0.376, criterion="Underbelly Rat"}

```

Comment:
This critter may be in the sewers, which is a different floor than the default.

Reply:
Should I fix it ?

Reply:
Yes, especially if there are other achievements in Dalaran Sewers.

Reply:
The Underbelly has uiMapId 126 ;) and map (old) name is same

Reply:
but floor 2, so I also add one to just Dalaran.